catalog.html

```diff
@@ -3,48 +3,55 @@
-    <form action="https://echo.htmlacademy.ru" method="GET">
-     <label for="search">
-      Поиск<input id="search" type="search"> <!--пока непонятно, как корректно задать эту форму и почему она не отображается-->
-     </label>
-    </form>
-     <a href="#">Закладки:</a>
-     <a href="#">Корзина: 10</a>
-     <a href="#">Оформить заказ</a>
-    <div>
+   <header class="main-header"> 
+     <div class="top-menu">
+       <a class="logo-top"><img src="img/logo.svg" width="220" height="47" alt="Логотип Техномарт" ></a>
+       <form action="https://echo.htmlacademy.ru" method="GET">
+          <label for="search">
+            Поиск
+            <input id="search" type="search"> <!--пока непонятно, как корректно задать эту форму и почему она не отображается-->
+          </label>
+        </form>
+       <a class="active-top-menu" href="#">Закладки:</a>
+       <a class="top-menu-basket active-top-menu" href="#">Корзина:10</a>
+       <a href="#"><span>Оформить заказ</span></a>
+      </div>
      <h1>
       Интернет-магазин<br> 
       строительных материалов<br> 
       и инструментов для ремонта
      </h1>
     </div>
-    <div>
-      <a href="tel:+78125550555">+7 (812) 555-05-55</a>
-      <address>г.Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</address>
+    <div class="company-info">
+     <a href="tel:+78125550555">+7 (812) 555-05-55</a>
+     <address>г.Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</address>
+    </div>
+    <div class="login-user">
+      <a href="#">Равшан Джамшутович</a>
+      <a href="#"><span class="visually-hidden">Выход из личного кабинета</span></a>
     </div>
-    <a href="#">Равшан Джамшутович</a>
-    <a href="#"><span class="visually-hidden">Выход из личного кабинета</span></a>
-    <ul>
-     <li><a href="#">Мои заказы</a>
-     <li><a href="#">Личный кабинет</a>
+    <ul class="info-user">
+      <li><a href="#">Мои заказы</a>
+      <li><a href="#">Личный кабинет</a>
     </ul>
-    <ul>
-     <li><a href="index.html">Главная</a></li>
-     <li><a href="#">Компания</a></li> <!--ссылка на раздел о Компании-->
-     <li><a>Каталог</a></li>  <!--ссылка в Каталог-->
-     <li><a href="#">Новости</a></li>  <!--ссылка на раздел Новости-->
-     <li><a href="#">Спецпредложения</a></li> <!--ссылка на раздел Спецпредложения-->
-     <li><a href="#">Доставка</a></li> <!--ссылка на раздел Доставка-->
-     <li><a href="#">Контакты</a></li> <!--ссылка на раздел Контакты-->
+    <ul class="menu-navigation">
+     <li><a>Главная</a></li>
+     <li><a href="#"><span>Компания</span></a></li>
+     <li><a href="catalog.html">Каталог</a></li> 
+     <li><a href="#">Новости</a></li> 
+     <li><a href="#">Спецпредложения</a></li>
+     <li><a href="#">Доставка</a></li> 
+     <li><a href="#">Контакты</a></li>
     </ul>
    </header>
    <main>
-     <h1>Перфораторы</h1>
-     <ul>
+     <h1><span class="main-heading">Перфораторы</span></h1>
```

Comment:
Также `span` здесь не нужен. Делай проще:
```suggestion
     <h1 class="main-heading">Перфораторы</h1>
```